assets/js/Containers/FrevaGPT/components/PendingAnswerComponent.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+
+import { Col, Card, Spinner, Accordion, Row } from "react-bootstrap";
+
+import Markdown from "react-markdown";
+
+import Highlight from "react-highlight";
+import "highlight.js/styles/atom-one-light.css";
+
+import * as constants from "../constants";
+
+function PendingAnswerComponent(props) {
+  const [renderedCode, setRenderedCode] = useState("");
+
+  useEffect(() => {
+    const parsedCode = renderCode(props.content);
+    if (parsedCode !== "") setRenderedCode(parsedCode);
+  }, [props.content]);
+
+  function renderCode(rawCode) {
+    let jsonCode = "";
+    let codeSnippets = "";
+
+    if (!rawCode.endsWith('"}')) jsonCode = rawCode + '"}';
```

Comment:
Coding style: You are using one-line if-statements without braces too often. Especially with some python-code on the side this can lead to hard to find errors and, therefore, we should always use braces. I would suggest, that we enforce the usage of braces in `.eslintrc` like this:

```
    "curly": [
      "warn",
      "all"
    ],
```

Apparently there are two occasions of missing braces in the databrowser as well (which would be my fault, I guess), you can fix them, too, while you are at it.